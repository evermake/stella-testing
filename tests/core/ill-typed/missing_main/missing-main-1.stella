language core;

fn returnTrue(b : Bool) -> Bool {
  return true
}
fn returnFalse(b : Bool) -> Bool {
  return false
}
