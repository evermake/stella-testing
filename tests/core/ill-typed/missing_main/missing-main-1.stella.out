TYPE ERROR:
Type Error Tag: [ERROR_MISSING_MAIN]
main function is not found in the program